src/app/components/menubar/menubar.ts

```diff
@@ -333,6 +334,7 @@ export class MenubarSub implements OnInit, OnDestroy {
     }
 
     onItemMouseLeave() {
+        this.activeItemPath = [];
         this.menubarService.mouseLeaves.next(true);
     }
```

Comment:
Untested,   but shouldn't the contents of the onItemMouseLeave be changed to this:

```
    onItemMouseLeave() {
        if (this.autoHide) {
             this.activeItemPath = [];
             this.menubarService.mouseLeaves.next(true);
        }
    }
```